### DISCUSSION TITLE
Add information to dwca export failure when traceback is suppressed.

### DISCUSSION THREAD

**Discussion by benanhalt** [specify/specify7#5521] [Feature Requests]
The traceback field in the dwca-export-failed message is suppressed when DEBUG is false. It should be replaced with a hint to turn on DEBUG.

**Comment by grantfitzsimmons** [comment]
This continues to be a pain point to this day

**Comment by maxpatiiuk** [comment]
Potential duplicate of #692

**Comment by grantfitzsimmons** [comment]
Not a duplicate of #692
